### PR TITLE
Update restore from backups guide

### PR DESCRIPTION
**Pull request by chao-xian**
Re-arranging, re-formatting and simplifying the page as I followed the
instructions. Should be a bit easier for someone who may be trying to
read it at 4am in the morning to follow.

### CODE REVIEW

File: source/manual/offsite-backup-and-restore.html.md [hunk 1]
```diff
@@ -14,179 +14,184 @@ It hasn't been reviewed for accuracy yet.
 [View history in old opsmanual](https://github.com/alphagov/govuk-legacy-opsmanual/tree/master/2nd-line/offsite-backup-and-restore.md)
```
Comment (tijmenb) [inline source/manual/offsite-backup-and-restore.html.md]: This warning can be removed now I think.

File: source/manual/offsite-backup-and-restore.html.md [hunk 2]
```diff
@@ -14,179 +14,184 @@ It hasn't been reviewed for accuracy yet.
...
-installing using apt-get:
+      ```
+      duplicity restore --file-to-restore data/backups/whitehall-mysql-backup-1.backend.publishing.service.gov.uk/var/lib/automysqlbackup/latest.tbz2 s3://s3-eu-west-1.amazonaws.com/govuk-offsite-backups-production/govuk-datastores/ /tmp/latest.tbz2
+      ```
+  * When this completes you may see the following 'error':
 
-```
-sudo apt-get install duplicity python-pip python-boto mysql-server
-```
+        ```
```
Comment (tijmenb) [inline source/manual/offsite-backup-and-restore.html.md]: Looks like the rendered version is showing the <code>```</code>, but not sure what the problem is:

https://github.com/alphagov/govuk-developer-docs/blob/cb1b6de7755754dbade34fa44eaabcf7e219d3d5/source/manual/offsite-backup-and-restore.html.md

Comment (chao-xian) [inline source/manual/offsite-backup-and-restore.html.md]: I've been going by how Middleman has been rendering https://govuk-tech-docs-pr-260.herokuapp.com/manual/offsite-backup-and-restore.html which looks ok.

Comment (tijmenb) [inline source/manual/offsite-backup-and-restore.html.md]: 👍

**Pull request merged**